requirements/production.txt

```diff
@@ -4,5 +4,6 @@
 
 MySQL-python==1.2.5	# GPL License
 PyYAML==3.11		# MIT
+gevent==1.0.2
```

Comment:
This was required for the ansible deploys when we moved to edx_django_service role, we're now using gevent worker w/ gunicorn